### DISCUSSION TITLE
Not generating transcripts.

### DISCUSSION THREAD

**Discussion by Gingseng17** [Dadangdut33/Speech-Translate#84] [Q&A]
Hi All, 
After I ran the transcription process, Speech-Translate does not generate any of the transcript files (srt, vtt, txt).  So I don't get anything at all.  Does anyone have encountered similar situation and may have some insights how to resolve this?  It was able to produce these reports before. 

Thanks

**Comment by joebinglab** [comment]
Suddenly I can't transcribe either as of two days ago... I have not changed any settings.. I normally transcribe about 10+ videos a day so not sure what's happened. Windows 10 did an update despite me having all those settings off, but other than that, not sure what else could've changed.

**Reply by joebinglab** [reply]
I'm running the latest version so can't even try "updating" it..

**Reply by joebinglab** [reply]
I think I found the problem... and I'm not sure if this will help you or not... but I was trying to transcribe videos that I had recently downloaded from YouTube using Python/YT-DLP. It looks like YouTube has changed it's download formats; and so downloading the mp4 format (with YT-DLP) no longer includes audio. I have to download the audio files separately. Very annoying and probably deliberate knowing YouTube.

**Reply by Gingseng17** [reply]
Thanks joebinglab.  I don't really download from youtube, so probably it's a different problem. But thanks for giving the heads up though.
